arraycontext/impl/pytato/__init__.py

```diff
@@ -54,10 +54,13 @@
 import abc
 import sys
 from collections.abc import Callable
+from dataclasses import dataclass
 from typing import TYPE_CHECKING, Any
 
 import numpy as np
 
+import pyopencl as cl
```

Comment:
`pyopencl` should not be imported top level because it creates a hard dependency (which at least most other places try to avoid?)

Reply:
Good catch, thanks!